CMake/SENSEIConfig.cmake.in

```diff
@@ -7,12 +7,10 @@ set(SENSEI_VERSION_DEVEL "@SENSEI_VERSION_DEVEL@")
 include(CMakeFindDependencyMacro)
 
 if (NOT SENSEI_DIR)
-  if (@SENSEI_BUILD@)
-    set(SENSEI_DIR "@CMAKE_BINARY_DIR@")
-  else()
-    set(SENSEI_DIR "@CMAKE_INSTALL_PREFIX@")
-  endif()
+  # Use relocatable lookup for SENSEI_DIR
+  set(SENSEI_DIR ${CMAKE_CURRENT_LIST_DIR}/../../)
```

Comment:
@kwryankrattiger the code you're replacing here captures the build location or install prefix so that all that is needed to configure a SENSEI instrumented simulation is our CMake config. On the other hand, the change you've made here does not do that since neither build nor install is guaranteed to be 2 levels up from the current directory. Perhaps this works for your use case, but the assumption does not in general hold. please find a solution that doesn't change the original functionality.

Reply:
Good call. I will add a new flag to the cmake config to enable this option. It will be disabled by default.